### DISCUSSION TITLE
Changing value with JS

### DISCUSSION THREAD

**Discussion by coder2000** [brianvoe/slim-select#322] [Q&A]
Does the value in SlimSelect change when the underlying select value changes? or is there a way to change the selected value through JS?

**Comment by barmillion** [comment]
yes you can do something like:
document.querySelector('select[name="session"]').slim.addData({value: '2022-09-24T04:00:00.000Z', text: 'Friday Sep 23, 2022 at 9:00 PM PDT'});
and:
document.querySelector('select[name="session"]').slim.set('2022-09-23T17:00:00.000Z');

**Comment by MilenFrom** [comment]
You can pass value through js, yes.
```
var select = new SlimSelect({
  select: '#select'
})
select.set('value')
// or
select.set(['value1', 'value2'])
```
